notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>NIT: comares -&gt; compares</p><p>NIT: name the link [Miolane et al 2020.]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>